api/controllers/models/taxonomy.js

```diff
@@ -1,10 +1,10 @@
 class Taxonomy {
```

Comment:
very minor, but this file has inconsistent use of semicolons.

Reply:
done